metadata_backend/api/health.py

```diff
@@ -0,0 +1,43 @@
+"""Handle health check endpoint."""
+import json
+from typing import Dict, Union
+
+from aiohttp import web
+from aiohttp.web import Request, Response
+from motor.motor_asyncio import AsyncIOMotorClient
+from pymongo.errors import ConnectionFailure
+
+from ..helpers.logger import LOG
+
+
+class HealthHandler:
+    """Handler for health check."""
+
+    async def get_health_status(self, req: Request) -> Response:
+        """Check health status of the application and return a JSON object portraying the status.
+
+        :param req: GET request
+        :returns: JSON response containing health statuses
+        """
+        db_client = req.app["db_client"]
```

Comment:
to avoid the 15 retry delay can we create a new AsyncIOMotorClient that uses the `url` from config ?

Reply:
Attempted this in the latest commits